### PR TITLE
Create rtptc.json

### PR DESCRIPTION
**Pull request by FeyrBert**

### CODE REVIEW

**Comment by Kamalisk** [comment]
invalid json, missing commas and quotes.

**Comment by FeyrBert** [comment]
Found one comma and one quote missing, should be fixed now.
Sorry about that

File: translations/it/pack/return/rtptc.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    },
+    {
+        "code": "52006",
+        "name": "Suggestione",
+        "text": "Utilizzo (3 cariche). Se non ci sono cariche sulla Suggestione, scartala.\n[action]Esaurisci la Suggestione: <b>Eludi</b>. Aggiungi il tuo valore di [willpower] al tuo valore di abilità in questo tentativo di elusione. Se non hai successo di 2 o più punti, rimuovi 1 carica dalla Suggestione.",
+        "traits": "Incantesimo.",
+        "slot": "Arcano"
+    },
+    {
+        "code": "52007"
```
Comment (Kamalisk) [inline translations/it/pack/return/rtptc.json]: still missing one here :P 
If you look at the diff here on hithub you can see the errors that there are errors highlighted in red (usually means there is some invalid json just before it)

Comment (FeyrBert) [reply translations/it/pack/return/rtptc.json]: Well, I'm blind, or dumb, or both xD
I'm always missing something, sorry about that

**Pull request merged**